openmicroscopy/bioformats
# OME-XML samples: synchronization and cleanup

See https://trello.com/c/visx1plk/36-schema-samples-migration. This PR supersedes https://github.com/openmicroscopy/bioformats/pull/2178/ and does the following:

- removes all non OME-XML samples from `components/specification/samples`
- synchronizes all OME-XML samples under `components/specification/samples` from the [live samples](http://downloads.openmicroscopy.org/images/OME-XML/) including naming changes and last additions and fixes
- fixes the OME-XML samples to comply with the policy decided on https://github.com/openmicroscopy/bioformats/pull/2286#issuecomment-197911813 using `XMLLINT_INDENT="  " xmllint -format`
- fixes the name of the samples across the repository especially in the various unit tests
- fixes the `specification` target validating the samples offline 

To test this PR:
- check all CI jobs are green including the Bio-Formats unit tests and the OME-XML C++ unit tests
- review the indentation changes in the OME-XML samples
- run the automated tests against the samples using the configuration files under `curated/ome-xml/ome-schemas`

Once this PR is accepted, the `samples` subfolder can be the home of the OME-XML samples, i.e. modifications/additions of the OME-XML samples can be proposed by PR. Once accepted, the data  under `data_repo/curated` should be updated to reflect these changes.
